content/zh-cn/blog/_posts/2021-11-08-steering-committee-results-2021.md

```diff
@@ -127,7 +127,7 @@ You can see what the Steering Committee meetings are all about by watching past
 ---
 
 <!--
-_This post was written by the [Upstream Marketing Working Group](https://github.com/kubernetes/community/tree/master/communication/marketing-team#contributor-marketing). If you want to write stories about the Kubernetes community, learn more about us._
+_This post was written by the [Upstream Marketing Working Group](https://github.com/kubernetes/community/tree/master/communication/contributor-comms#contributor-marketing). If you want to write stories about the Kubernetes community, learn more about us._
 -->
-_本文是由[上游营销工作组](https://github.com/kubernetes/community/tree/master/communication/marketing-team#contributor-marketing)撰写的。
+_本文是由[上游营销工作组](https://github.com/kubernetes/community/tree/master/communication/contributor-comms#contributor-marketing)撰写的。
```

Comment:
In k/website project, we will limit every PR to one language.

We could open another PR once this is reviewed and merged. Or leave it to zh-cn contributors.